python/demo-osc-calls.sh

```diff
@@ -6,19 +6,25 @@
 oscsend localhost 5006 /paths
 
 # Add an effect, trivial case
-oscsend localhost 5006 /rosary/add_effect s bounce
+oscsend localhost 5006 /rosary/add_effect/name/bounce
```

Comment:
The pattern is now /rosary/<METHOD_NAME>/<ARG1_NAME>/<ARG1_VAL>/<ARG2_NAME>/<ARG2_VAL>, etc.

Which means positional arguments are no longer accepted